code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -285,3 +285,22 @@
 		update_icon()
 
 		return FALSE
+
+/obj/item/weapon/material/brassknuckles/gold
+    default_material="gold"
+    name = "Brass Knuckles"
+    desc = "Brass Knuckles used on your fist for hard hits used in gangs"
```

Comment:
No capitalization please here or in the name title, the grammar also reads off strangely.

``,often used for gang warfare`` would be a suitable fix.``